<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartWaterGUI</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>561</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabSensorsMeas">
     <attribute name="title">
      <string>Текущие измерения</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>471</width>
        <height>431</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="layoutSensorsMeas" stretch="0,0,0,0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutSensor_1">
         <item>
          <widget class="QCheckBox" name="checkBoxSensor_1">
           <property name="text">
            <string>Гнездо 1</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutSensor_1">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QComboBox" name="comboBoxSensor_1">
             <item>
              <property name="text">
               <string>Датчик температуры</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="dataMeasSensor_1">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelUnitsSensor_1">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>℃</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutSensor_2">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QCheckBox" name="checkBoxSensor_2">
           <property name="text">
            <string>Гнездо 2</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutSensor_2">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QComboBox" name="comboBoxSensor_2">
             <item>
              <property name="text">
               <string>Датчик рН</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="dataMeasSensor_2">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelUnitsSensor_2">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>ед. рН</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutSensor_3">
         <item>
          <widget class="QCheckBox" name="checkBoxSensor_3">
           <property name="text">
            <string>Гнездо 3</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutSensor_3">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QComboBox" name="comboBoxSensor_3">
             <item>
              <property name="text">
               <string>Датчик проводимости</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="dataMeasSensor_3">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelUnitsSensor_3">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>мкСм</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutSensor_4">
         <item>
          <widget class="QCheckBox" name="checkBoxSensor_4">
           <property name="text">
            <string>Гнездо 4</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutSensor_4">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QComboBox" name="comboBoxSensor_4">
             <item>
              <property name="text">
               <string>Датчик кислорода</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="dataMeasSensor_4">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelUnitsSensor_4">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>%</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutSensor_5">
         <item>
          <widget class="QCheckBox" name="checkBoxSensor_5">
           <property name="text">
            <string>Гнездо 5</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutSensor_5">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QComboBox" name="comboBoxSensor_5">
             <item>
              <property name="text">
               <string>Датчик ОВП</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="dataMeasSensor_5">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelUnitsSensor_5">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>мВ</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutSensor_6">
         <item>
          <widget class="QCheckBox" name="checkBoxSensor_6">
           <property name="text">
            <string>Гнездо 6</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayoutSensor_6">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QComboBox" name="comboBoxSensor_6">
             <item>
              <property name="text">
               <string>Датчик мутности</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="dataMeasSensor_6">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelUnitsSensor_6">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>NTU</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QRadioButton" name="radioButtonSW">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Smart Water</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonSWIons">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Smart Water Ions</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabCalibration">
     <attribute name="title">
      <string>Калибровка</string>
     </attribute>
     <widget class="QComboBox" name="boxSensors">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelSensors">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>16</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Датчик</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCalibrationSolution">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Калибровочный раствор</string>
      </property>
     </widget>
     <widget class="QComboBox" name="boxCalibrationSolution">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>50</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonStartCalibration">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Начать калибровку</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelStabilisationTime">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Время стабилизации</string>
      </property>
     </widget>
     <widget class="QComboBox" name="boxStabilisationTime">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>90</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1 минута</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2 минуты</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3 минуты</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4 минуты</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5 минут</string>
       </property>
      </item>
     </widget>
     <widget class="QProgressBar" name="progressBarCalibration">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>541</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textCalibrationValues">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>161</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelCalibrationValues_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Калибровочные</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCalibrationValues_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>значения:</string>
      </property>
     </widget>
     <widget class="PlotWidget" name="graphicsViewCalibration" native="true">
      <property name="geometry">
       <rect>
        <x>179</x>
        <y>209</y>
        <width>371</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabFirmwareInfo">
     <attribute name="title">
      <string>Информация об устройстве</string>
     </attribute>
     <widget class="QLabel" name="labelDeviceInfo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Информация об устройстве</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>481</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutDeviceInfo">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutDeviceName">
         <item>
          <widget class="QLabel" name="labelDeviceName">
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Имя устройства:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="dataDeviceName">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutSerialID">
         <item>
          <widget class="QLabel" name="labelSerialID">
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Serial ID:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="dataSerialID">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutFirmware">
         <item>
          <widget class="QLabel" name="labelFirmware">
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Прошивка:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="dataFirmware">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutFirmwareVersion">
         <item>
          <widget class="QLabel" name="labelFirmwareVersion">
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Версия прошивки:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="dataFirmwareVersion">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutmd5">
         <item>
          <widget class="QLabel" name="labelmd5">
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>md5 hash:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="datamd5">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="labelAppVersion">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>570</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SmartWaterGUI версия ПО 2.0</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="70,200,40,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="0" column="4">
      <widget class="QLabel" name="dataStatus">
       <property name="text">
        <string>не подключено</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelUSBPort">
       <property name="text">
        <string>USB Порт</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="boxUSBPorts">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="labelStatus">
       <property name="text">
        <string>Статус:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelBatteryUnits">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>570</y>
      <width>16</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dataBattery">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>570</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelBattery">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>570</y>
      <width>47</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Заряд:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
